src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/config.jelly

```diff
@@ -27,7 +27,9 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
     <j:set var="jobFieldId" value="${h.generateId()}"/>
```

Comment:
In general I think we should be able to get rid of `h.generateId()` in these refactorings, by making the behavior function take the element argument into account and looking for siblings. Same for the hard-coded id `params`, which we did not even bother uniquifying using `generateId` since we expect this Jelly facet to be loaded only once in a page (**Snippet generator**).

Reply:
Oh makes sense, I want to contribute to Jenkins in long term as well so I can work on it as a separate issue?
Also is there any slack for Jenkins or is Gitter the preferred platform?

Reply:
> I can work on it as a separate issue?

Sure, if you would like. It is not a priority, would just be a nice stylistic improvement.

> is there any slack for Jenkins or is Gitter the preferred platform?

https://www.jenkins.io/chat/ though TBH I am not normally on those. Not sure why Slack is not used.